charts/lfx-platform/templates/openfga/model.yaml

```diff
@@ -45,7 +49,21 @@ spec:
           relations
             define project: [project]
             define committee: [committee]
-            define organizer: [user]
-            define participant: [user, committee#member]
-            define viewer: [user:*, committee#member] or participant or auditor from project
+            # The organizer relation identifies a user who can manage this one meeting.
+            # That means they can update the meeting details, invite/uninvite participants, etc.
+            define organizer: [user] or meeting_organizer from project or writer from project
+            # The host relation identifies a user who is a host of this meeting.
+            # This is different than the organizer relation because an organizer isn't necessarily
+            # the user who is hosting the meeting, nor is the host necessarily the one who is
+            # organizing the meeting. For example, a host may need to retrieve the Zoom host key
+            # but shouldn't be able to update the meeting details.
+            define host: [user]
+            # The participant relation identifies a user who is a participant in this meeting.
+            # This can either mean they are invited to the meeting or they attended the meeting
+            # without being invited. In either case, they are a participant of that meeting.
+            define participant: [user] or host or member from committee
```

Comment:
I'm on the fence about including "host" here ... I think it will depend on the data models. If we assume that, in the data, you don't add a person as both a host _and_ a participant, only as one, then we need this relation. But if "host" is modeled in the data as a *flag* on the participant object, or there is a constraint that prevents adding someone to a list of hosts without them first being added to the list of participants, then we don't need this relation here (and we may get a slight performance boost from removing unneeded relationship).